server/server.go

```diff
@@ -200,26 +199,17 @@ func (server *HTTPServer) readRuleForReport(writer http.ResponseWriter, request
 		return
 	}
 
-	report, err := server.Storage.ReadRuleForReport(orgID, clusterName, ruleID, errorKey)
+	templateData, err := server.Storage.ReadSingleRule(orgID, clusterName, ruleID, errorKey)
 	if err != nil {
 		log.Error().Err(err).Msg("Unable to read rule report for cluster")
 		handleServerError(writer, err)
 		return
 	}
 
-	var ruleDetails json.RawMessage
-	err = json.Unmarshal([]byte(report), &ruleDetails)
-	if err != nil {
-		log.Error().Err(err).Msg("Unable to parse cluster rule report")
-		handleServerError(writer, err)
-		return
-	}
-
-	ruleIDSplitted := strings.Split(string(ruleID), "|")
 	reportRule := types.RuleOnReport{
-		TemplateData: string(ruleDetails),
-		Module:       types.RuleID(ruleIDSplitted[0]),
-		ErrorKey:     types.ErrorKey(ruleIDSplitted[1]),
+		TemplateData: templateData,
+		Module:       ruleID,
+		ErrorKey:     errorKey,
```

Comment:
So it's `rule_key` or `error_key`?

Reply:
I didn't rename it everywhere (yet), renaming stuff in code is easy thing, renaming sql requires a migration.